doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -73,6 +73,10 @@ Matter
 
   * Support for Wi-Fi Network Diagnostic Cluster (which counts the number of packets received and transmitted on the Wi-Fi interface).
 
+* Updated:
+
+  * Default heap implementation to use Zephyr's `sys_heap` of size 10kB (:kconfig:option:`CONFIG_CHIP_MALLOC_SYS_HEAP`) to better control RAM usage of Matter applications.
```

Comment:
```suggestion
  * Default heap implementation to use Zephyr's ``sys_heap`` size of 10 kB (:kconfig:option:`CONFIG_CHIP_MALLOC_SYS_HEAP`) to better control the RAM usage of Matter applications.
```

Reply:
Removed the size information as the main information I wanted to convey is that we switch to Zephyr's `sys_heap` implementation (from a different implementation). I don't want to make the impression that we just change the size.